chrome/content/zotero/itemTree.jsx

```diff
@@ -59,7 +63,8 @@ var ItemTree = class ItemTree extends LibraryTree {
 		dragAndDrop: false,
 		persistColumns: false,
 		columnPicker: false,
-		columns: COLUMNS,
+		// By default, use the global columns
+		columns: null,
```

Comment:
Why this change?

Reply:
Oh, I forgot this part. This is brought by the previous versions.